docs/TuxOwneeClass.md

```diff
@@ -0,0 +1,250 @@
+As with any other `React` class, a [render](#http://facebook.github.io/react/docs/component-specs.html#render) method is required. This method is responsible for returning a `React Element` which can be rendered onto the DOM.
+
+```javascript
+    owneeClassProps.render = function () {
+      return (
+        <div>
+          <h1>hello world</h1>
+        </div>
+      );
+    };
+```
+
+##### <a id="owneeClassProps-LIFECYCLE_AND_SPECS"></a>Methods/Properties - `owneeClassProps.LIFECYCLE_AND_SPECS` - type: FUNCTIONs - optional [#](#owneeClassProps-LIFECYCLE_AND_SPECS)
+As with any other `React` class, the `owneeClassProps` can optionally include `React` [lifecycle methods and specs](#http://facebook.github.io/react/docs/component-specs.html).
+
+```javascript
+    owneeClassProps.componentDidMount = function () {
+      roomActions.get();
+    };
+```
+
+##### <a id="owneeClassProps-nearestOwnerProps"></a>Property - `owneeClassProps.nearestOwnerProps` [#](#owneeClassProps-nearestOwnerProps)
+Allows the `Ownee` to access the `nearestOwnerProps` on the closest `OwnerClass`.
+nearestOwnerProps - should be static, if you need to pass in dynamic props use standard method (works well for an example)
+
+**NOTE** `nearestOwnerProps` should be treated as immutable just like standard component props. Thus, you should never modify the `nearestOwnerProps` object. It is designed to store static methods and data.
+
+##### <a id="owneeClassProps-nearestOwnerPropTypes"></a>Property - `owneeClassProps.nearestOwnerPropTypes` - type: OBJECT - optional [#](#owneeClassProps-nearestOwnerPropTypes)
+Allows you to perform prop validation on the `nearestOwnerProps` to this `OwneeClass` component. For the full list of available properties that can be checked, please look at the [Reusable Components](http://facebook.github.io/react/docs/reusable-components.html) documenation.
+
+```javascript
+    owneeClassProps.nearestOwnerPropTypes = {
+      deleteMessage: React.PropTypes.function.isRequired,
+      editMessage: React.PropTypes.function.isRequired,
+    };
+```
+
+##### <a id="owneeClassProps-anyPropTypes"></a>Property - `owneeClassProps.anyPropTypes` - type: OBJECT - optional [#](#owneeClassProps-anyPropTypes)
+Allows you to perform prop validation on both the standard props of a component, as well as those props available via `nearestOwnerProps`. For the full list of available properties that can be checked, please look at the [Reusable Components](http://facebook.github.io/react/docs/reusable-components.html) documenation.
+
+```javascript
+    owneeClassProps.anyPropTypes = {
+      message: React.PropTypes.object.isRequired,
+      deleteMessage: React.PropTypes.function.isRequired,
+      editMessage: React.PropTypes.function.isRequired
+    };
+```
+
+**NOTE** When performing prop validations with `anyPropTypes` that props in `this.props` are validated first, so they will override the information within nearestOwnerProps.
```

Comment:
Change to: ...so they will override the validation of the `nearestOwnerProps`. For example, in the above scenario if `this.props.message` is an `object`, but `this.nearestOwnerProps.message` is a `string`, then the `message` property will be treated as valid since `this.props` takes priority.